src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -1502,18 +1438,25 @@ RESULT_CODE ProtocolHandlerImpl::HandleControlMessageStartSession(
   return RESULT_OK;
 }
 
+// DEPRECATED
 void ProtocolHandlerImpl::NotifySessionStartedResult(
     int32_t connection_id,
     uint8_t session_id,
     uint8_t generated_session_id,
     uint32_t hash_id,
     bool protection,
+    std::vector<std::string>& rejected_params) {}
```

Comment:
For safety, just use this deprecated method as a wrapper function:
```
  // DEPRECATED
  void ProtocolHandlerImpl::NotifySessionStartedResult(
      int32_t connection_id,
      uint8_t session_id,
      uint8_t generated_session_id,
      uint32_t hash_id,
      bool protection,
      std::vector<std::string>& rejected_params) {
    protocol_handler::StartSessionContext context(
        connection_id,
        session_id,
        generated_session_id,
        0,
        hash_id,
        protection);
    NotifySessionStarted(context, rejected_params);
  }
```

Reply:
@jacobkeeler added wrapper to deprecated function. See https://github.com/smartdevicelink/sdl_core/commit/324b740fa4a8d976b4e47db5d53bbf83659b4f8a